src/components/useDarkMode.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+export const useDarkMode = () => {
+const [theme, setTheme] = useState('light');
+const [componentMounted, setComponentMounted] = useState(false);
+
+const setMode = mode => {
+    window.localStorage.setItem('theme', mode)
+    setTheme(mode)
+};
+
+const toggleTheme = () => {
+    if (theme === 'light') {
+        setMode('dark');
+    } else {
+        setMode('light');
+    }
+};
+
+useEffect(() => {
+    const localTheme = window.localStorage.getItem('theme');
+    if (localTheme) {
```

Comment:
hey @teradaian 
 local theme === ?? , is it equal to either light or dark?  what is it defaulted to?

Reply:
it's reading local storage and checking if 'theme' is present (as it default checks as boolean without the second argument). The idea is to make sure it stays on dark mode or light mode when selected vs resetting on reload etc.